test/e2e/cluster_extension_install_test.go

```diff
@@ -268,22 +268,18 @@ func TestClusterExtensionInstallRegistry(t *testing.T) {
 			t.Log("By eventually reporting a successful resolution and bundle path")
 			require.EventuallyWithT(t, func(ct *assert.CollectT) {
 				assert.NoError(ct, c.Get(context.Background(), types.NamespacedName{Name: clusterExtension.Name}, clusterExtension))
-				assert.Equal(ct,
-					&ocv1alpha1.ClusterExtensionResolutionStatus{Bundle: ocv1alpha1.BundleMetadata{
-						Name:    fmt.Sprintf("%s-operator.1.2.0", tc.packageName),
-						Version: "1.2.0",
-					}},
-					clusterExtension.Status.Resolution,
-				)
+				cond := apimeta.FindStatusCondition(clusterExtension.Status.Conditions, ocv1alpha1.TypeProgressing)
+				require.NotNil(ct, cond)
+				assert.Equal(ct, metav1.ConditionTrue, cond.Status)
+				assert.Equal(ct, ocv1alpha1.ReasonSuccess, cond.Reason)
+				assert.Contains(ct, cond.Message, "resolved to")
```

Comment:
We already have a check for the progressing condition immediately below this. We can probably remove this added code.

I think the failure you may be seeing in the e2e is because the check is looking for Progressing to be "True", but a successful reconciliation should actually result in Progressing set to "False".

Reply:
Fixed in https://github.com/operator-framework/operator-controller/commit/febe76e179455d20c593b3f5956a770835afcedc